GitUI/ScriptsEngine/SimplePrompt.cs

```diff
@@ -1,12 +1,16 @@
-﻿namespace GitUI.ScriptsEngine
+﻿using System.Runtime.CompilerServices;
+
+namespace GitUI.ScriptsEngine
 {
-    public partial class SimplePrompt : Form
+    internal partial class SimplePrompt : Form, IUserInputPrompt
     {
         public string UserInput { get; private set; } = "";
 
-        public SimplePrompt()
+        public SimplePrompt(string label, string defaultValue)
         {
             InitializeComponent();
+            txt_UserInput.Text = defaultValue;
+            Text = label;
```

Comment:
```suggestion
            if (!string.IsNullOrWhitespace(label))
            {
                Text = label;
            }
```

Reply:
The RegEx guarantees that there will be at least 1 character but why not if the user want to shoot in its own feet... (fixed)